libkbfs/keybase_daemon_local.go

```diff
@@ -495,3 +495,9 @@ func newKeybaseDaemonLocal(codec kbfscodec.Codec,
 		favoriteStore: favoriteStore,
 	}
 }
+
+// EstablishMountDir asks the service for the current mount path
```

Comment:
It's a stupid convention, but interface implementors in KBFS use comments of this form instead:

```go
// EstablishMountDir implements the KeybaseDaemon interface for KeybaseDaemonLocal.
```